packages/design-tokens/src/ThemeManager.ts

```diff
@@ -10,10 +10,9 @@ import { makeCssVariableDefinitionsMap } from "./lib/makeCssVariableDefinitionsM
 export class ThemeManager<Theme extends BaseTheme = BaseTheme> {
   private themeChangeListeners = [] as Array<(theme: Theme) => void>
   private theme: Theme
-  private rootElement = document.documentElement
+  private rootElement: HTMLElement | null = null
   constructor(
     theme: Theme,
-    rootElement = document.documentElement,
```

Comment:
thanks for the review @dougmacknz . Just a quick question, would this change be considered as a breaking changes since constructor doesn't take `rootElement` anymore? I did a search across our repos and always see just `theme` being provided but not the other two (except for ThemeManager which I did fixed)

Reply:
Good question. Technically the answer is probably yes, but having taken a look through all of the usages of `ThemeManager`, the only place that was passing a second arg was the one in `ThemeManager.spec` that you've already fixed up. So I wouldn't do a breaking change here, especially because it's the `design-tokens` package, which is used as a peer dependency in all of our Kaizen packages. 

Related: I think I would have labelled this PR a `feat` over a `fix` to trigger a minor version bump over a patch version. I don't feel strongly though and it doesn't have huge ramifications, but if you wanted to change it, all you would need to do at this point is adjust your PR title.

Reply:
awesome thanks for the quick response 👍 . I will update PR to `feat `